kube-client/src/client/auth/oidc.rs

```diff
@@ -481,7 +475,7 @@ mod tests {
         let invalid_claims_token = format!(
             "{}.{}.{}",
             token_valid.split_once('.').unwrap().0,
-            base64::encode(serde_json::to_string(&invalid_claims).unwrap()),
+            BASE64_ENGINE.encode(serde_json::to_string(&invalid_claims).unwrap()),
```

Comment:
This should also use regular b64, not b64url.

Reply:
I think actually this should use the JWT engine because it's encoding the `Claims` which is what we use the engine for.

This is a test though, and it passes either way.

Reply:
Ah gotcha, so actually both versions were buggy!